src/main/java/org/aion/wallet/ui/components/account/AccountCellItem.java

```diff
@@ -23,6 +25,10 @@
 
     private static final String ICON_DISCONNECTED = "/org/aion/wallet/ui/components/icons/icon-disconnected-50.png";
 
+    private static final String EDIT_ICON = "/org/aion/wallet/ui/components/icons/pencil-edit-button.png";
```

Comment:
let's try to keep a consistent naming scheme, at least in the same file - either rename the new constants or the old ones